src/Microsoft.Data.SqlClient/netcore/src/Microsoft/Data/SqlClient/SNI/SNICommon.cs

```diff
@@ -90,8 +90,7 @@ public void Write(Span<byte> bytes)
     /// <summary>
     /// SMUX packet flags
     /// </summary>
-    [Flags]
```

Comment:
The MC-SMP standard states that these cannot be combined. They aren't being combined in the code but I chose to prevent possible future confusion and remove the attribute.